include/deal.II/grid/tria_accessor.templates.h

```diff
@@ -2233,9 +2238,10 @@ TriaAccessor<0, dim, spacedim>::operator ++ () const
 template <int dim, int spacedim>
 inline
 void
-TriaAccessor<0, dim, spacedim>::operator -- () const
+TriaAccessor<0, dim, spacedim>::operator -- ()
 {
-  Assert (false, ExcNotImplemented());
+  if (global_vertex_index!=numbers::invalid_unsigned_int)
+    --global_vertex_index;
```

Comment:
I don't understand what you mean. How is it an accident that it works? We defined ```numbers::invalid_unsigned_int``` to be ```(unsigned int) -1```.